src/modules/RA/DataLayer/components/tabs/StyleTab/Style/WizardPiechart.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import { useTranslate, RadioButtonGroupInput } from 'react-admin';
+import { makeStyles } from '@material-ui/core/styles';
+import FormLabel from '@material-ui/core/FormLabel';
+
+import PiePieceStyleField from './PiePieceStyleField';
+
+import styles from './styles';
+import PieChartRadiusStyleField from './PieChartRadiusStyleField';
```

Comment:
```suggestion
import PiePieceStyleField from './PiePieceStyleField';
import PieChartRadiusStyleField from './PieChartRadiusStyleField';

import styles from './styles';
```

*(grouping components)*